build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+# Releases this Puppet module to the Puppet Forge
```

Comment:
Comment should be that it's building Puppet module, not releasing?

Reply:
ack, you're right

Reply:
updated.